common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -2,42 +2,69 @@
 import unittest
 
 from xmodule import graders
-from xmodule.graders import Score, aggregate_scores
+from xmodule.graders import ProblemScore, AggregatedScore, aggregate_scores
 
 
 class GradesheetTest(unittest.TestCase):
-    '''Tests the aggregate_scores method'''
+    """
+    Tests the aggregate_scores method
+    """
 
     def test_weighted_grading(self):
         scores = []
-        Score.__sub__ = lambda me, other: (me.earned - other.earned) + (me.possible - other.possible)
+        ProblemScore.__sub__ = lambda me, other: (me.earned - other.earned) + (me.possible - other.possible)
```

Comment:
Good catch.  I also don't see where the `__sub__` came into play.  I removed it - and the tests still pass - so we're good.